airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/typing_deduping/SnowflakeSqlGenerator.java

```diff
@@ -577,6 +586,11 @@ public static String escapeSqlIdentifier(String identifier) {
     return escapeJsonIdentifier(identifier);
   }
 
+  private static String prefixReservedKeyword(final String columnName) {
```

Comment:
nit: this is only doing column names, rename to something like `prefixReservedColumnName`